realm/realm-library/src/main/java/io/realm/OrderedRealmCollection.java

```diff
@@ -137,4 +137,14 @@
      * @throws UnsupportedOperationException if the collection is unmanaged.
      */
     boolean deleteLastFromRealm();
+
+    /**
+     * Creates a snapshot from this {@link OrderedRealmCollection}.
```

Comment:
Perhaps add a bit more what a snapshot actually means, i.e. element positions not updating and even an iterator example would probably be nice.